File: Source/WickedSick.Fayde.Client.Engine/Typescript/Controls/Page.ts
/// <reference path="UserControl.ts" />
/// CODE

module Fayde.Controls {
    export class Page extends UserControl {
        static TitleProperty: DependencyProperty = DependencyProperty.Register("Title", () => String, Page);
        Title: string;
    }
    Nullstone.RegisterType(Page, "Page");
}

File: Source/Fayde.Server/Controls/Page.cs
using Fayde.Xaml.Metadata;

namespace Fayde.Controls
{
    public class Page : UserControl
    {
        public static readonly PropertyDescription TitleProperty = PropertyDescription.Register("Title", typeof(string), typeof(Page));
        public string Title
        {
            get { return GetValue("Title") as string; }
            set { SetValue("Title", value); }
        }
    }
}